tests/Integration/Behaviour/Features/Scenario/Carrier/carrier_management.feature

```diff
@@ -41,13 +60,23 @@ Feature: Carrier management
       | trackingUrl  | http://example.com/track.php?num=@ |
       | position     | 2                                  |
       | active       | true                               |
+      | max_width    | 1234                               |
+      | max_height   | 1234                               |
+      | max_depth    | 1234                               |
+      | max_weight   | 1234                               |
+      | group_access | visitor, guest                     |
```

Comment:
I you use the same values as the initial ones you're not testing the edition actually works